OPHD/UI/StringTable.cpp

```diff
@@ -32,7 +32,7 @@ void StringTable::draw(NAS2D::Renderer& renderer) const
 	}
 }
 
-void StringTable::position(NAS2D::Point<int> position)
+void StringTable::position(const NAS2D::Point<int>& position)
```

Comment:
For structs as small as `Point`, it's typically faster to pass by-value. That's the convention we've been using in most places for `Point` and `Vector`. Admittedly, we're not entirely consistent about it. As an added bonus, passing by-value tends to create less syntactic noise.

For larger structs, such as `Rectangle`, we do still tend to pass by `const&`. The difference in speed is not typically very large for structs of these small sizes, so this rule doesn't need to be set in stone. Typically the speed switch over for by-value versus by-reference passing of structs is at about 2-3 `int` sized elements.

Reply:
OK, note fix in PR #567 for Control.h. Will revert this change shortly.